Corona-Warn-App/src/main/res/drawable/contact_diary_onboarding_background.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="rectangle">
+            <solid android:color="@color/colorSurface1" />
+        </shape>
+    </item>
+</selector>
```

Comment:
I think we don't need a `selector` here if it is just a background or?

Reply:
Should be fine for now, but we can consider to remove it from all files where it is not needed